### DISCUSSION TITLE
formula with ID [Jun 10]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2421] [Discord Threads]
> 🙋‍♂️ **Damir**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### CONCAT s a string function, which means you cannot put ID field  
#  
> 🙋‍♂️ **wkw**  
#### there'd be some enhancements for formula  
#  
> 💁 **o1lab**  
#### You can track the issue here https://github.com/nocodb/nocodb/issues/2304   
####    
#### Thank you Wing  
#  
> 🙋‍♂️ **Damir**  
#### Ok, thanks a lot and keep on with good work  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/984412737734082561)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
